tests/test_item_resource.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import requests
+import re
+
+from urllib.parse import urljoin
+
+
+class TestItemResourceJson(object):
+    @pytest.fixture
+    def response(self, endpoint):
+        self.register_name = re.sub(r'http[s]?://([^\.]+)(.*)', r'\1', endpoint)
+
+        entry = requests.get(urljoin(endpoint, 'entry/1.json'))
+
+        item_hash = entry.json()['item-hash']
+
+        return requests.get(urljoin(endpoint, 'item/' + item_hash + '.json'))
+
+    def test_response_contents(self, response, register_register_endpoint ):
+        register_data = requests.get(
+            urljoin(register_register_endpoint, '/record/' + self.register_name + '.json'))
```

Comment:
I feel like this should be made available as a fixture as well at some point, since other tests will surely refer to register register data.